### PR TITLE
news and version bump

### PR DESCRIPTION
**Pull request by piever**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# AlgebraOfGraphics.jl v0.6 Release Notes
+
+## Breaking Changes
+
+- Default axis linking behavior has changed: now only axes corresponding to the same variable are linked.
```
[the remark this answers is not in the document]
Comment (piever) [reply NEWS.md:5]: So, for `layout` , it is still true that things are axes are only linked if they correspond to the same variable, but there is an extra breaking change in that AoG tries to also hide decorations by default?

**Comment by piever** [comment]
> do you mind reviewing #289 before tagging a new release?

That's non-breaking (it only changes internal functions and fixes a bug), so I imagine it can go in 0.6.1

It looks like a relative small change, but I'm a bit hesitant to introduce a new struct to work around a Makie bug, so I thought we should take our time before merging it. I was trying to figure out if that approach allows us to simplify other parts of the code base.

File: NEWS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (greimel) [reply NEWS.md:5]:
```suggestion
- Default axis linking behavior has changed: now only axes corresponding to the same variable are linked. For consistency with `row`/`col`, `layout` will hide inner axis decorations and span axis labels if appropriate.
```

[the remark this answers is not in the document]
Comment (piever) [reply NEWS.md:5]: Ah, that makes it much clearer, thanks!

**Pull request merged**